sqlglot/lineage.py

```diff
@@ -204,13 +213,15 @@ def to_node(
             source = scope.sources.get(table)
 
             if isinstance(source, Scope):
+                selected_node, selected_source = scope.selected_sources.get(table, (None, None))
```

Comment:
Maybe I'm missing something, but I think `selected_source` is unused?
```suggestion
                selected_node, _ = scope.selected_sources.get(table, (None, None))
```

Reply:
Ah, you're right.